### PR TITLE
feat: extract settings and common package

### PR DESCRIPTION
**Pull request by karolsojko**

### CODE REVIEW

File: packages/settings/src/Domain/Setting/SettingName.ts [hunk 1]
```diff
@@ -1,3 +1,3 @@
-export enum MfaSetting {
+export enum SettingName {
   MfaSecret = 'MFA_SECRET',
 }
```
Comment (gorjan5sk) [inline packages/settings/src/Domain/Setting/SettingName.ts:3]:
```suggestion
/**
 * Enumerate all Settings with their names as identifiers
 *
 * Extend by adding more enum members
 */
export enum Setting {
  MfaSecret = 'MFA_SECRET',
}

/**
 * Describe Setting payloads (contents) matched with Setting enum keys
 *
 * Extend by adding more properties with keys from Setting enum and value with the corresponding Payload
 */
export type SettingPayload = {
  [Setting.MfaSecret]: { secret: string };
};

```

Comment (gorjan5sk) [reply packages/settings/src/Domain/Setting/SettingName.ts:3]: You beat me to the refactor. What do you think about this ^?

File: packages/auth/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@standardnotes/auth",
-  "version": "3.2.0",
```
Comment (gorjan5sk) [inline packages/auth/package.json:3]: This would be a breaking change for the dependent `auth` repo. I think we should change the MINOR version at least?

**Review by vardan-arm — APPROVED** [review]

File: packages/settings/src/Domain/Setting/SettingName.ts (hunk 1 above)
Comment (karolsojko) [reply packages/settings/src/Domain/Setting/SettingName.ts:3]: hmm not sure if we need to define the payload like that - but feel free to add it another PR.

Additionally keep in mind to keep 1 type/class/interface per file

File: packages/auth/package.json (hunk 2 above)
Comment (karolsojko) [reply packages/auth/package.json:3]: good point

**Review by gorjan5sk — APPROVED** [review]

**Pull request merged**